apps/omg_watcher/lib/exit_processor/in_flight_exit_info.ex

```diff
@@ -215,6 +215,14 @@ defmodule OMG.Watcher.ExitProcessor.InFlightExitInfo do
     end
   end
 
+  def is_input_piggybacked?(%__MODULE__{} = ife, index) when is_integer(index) and index < @max_inputs do
```

Comment:
if we're positive about doing this refactor then maybe `is_piggybacked` can be private, and the verbose function be used in consumers of this module

Reply:
We need is_piggybacked to be public. There is a use case in core where we don't know if we are asking for tx inputs or outputs. See `with` in `finalize_in_flight_exits`.

Reply:
ok, nvm then, we can reassess that when we go about refactoring all this